openfe/analysis/plotting.py

```diff
@@ -82,24 +82,40 @@ def plot_lambda_transition_matrix(matrix: npt.NDArray) -> Axes:
             )
 
         # anotate axes
-        base_settings = {
+        base_settings: dict[str, Union[str, int]] = {
             'size': 10, 'va': 'center', 'ha': 'center', 'color': 'k',
             'family': 'sans-serif'
         }
         for i in range(num_states):
             ax.annotate(
-                i, xy=(i + 0.5, 1), xytext=(i + 0.5, num_states + 0.5),
+                text=f"{i}",
+                xy=(i + 0.5, 1),
+                xytext=(i + 0.5, num_states + 0.5),
+                xycoords='data',
```

Comment:
Adding the defaults is a bit much, but thought I might as well be explicit... thoughts?

Reply:
I like it!